feature.env

```diff
@@ -16,7 +16,7 @@ SYSTEM_CLIENT_SECRET=clientsecret
 SQS_POLLING_WAIT_TIME_MS=2000
 REDIS_HOST=localhost
 HARD_STOP_ENABLED=true
-
+TOKEN_VERIFICATION_ENABLED=true
```

Comment:
tests now use token verification, the health checks now check partial failure works as expected